test/int/sources/sources.noBreakpoints.test.ts

```diff
@@ -0,0 +1,43 @@
+import * as path from 'path';
+import * as testSetup from '../testSetup';
+import * as assert from 'assert';
+import { puppeteerSuite, puppeteerTest } from '../puppeteer/puppeteerSuite';
+import { TestProjectSpec } from '../framework/frameworkTestSupport';
+import { loadedSourcesContainsScript, createScriptItemFromInfo } from './sources.utils';
+
+// We are using the "Simple" project app inside the testdata directory
+const DATA_ROOT = testSetup.DATA_ROOT;
+const SIMPLE_PROJECT_ROOT = path.join(DATA_ROOT, 'simple');
+const TEST_SPEC = new TestProjectSpec( { projectRoot: SIMPLE_PROJECT_ROOT } );
+
+puppeteerSuite('Static HTML Pages Without Breakpoints', TEST_SPEC, (suiteContext) => {
+    puppeteerTest('Static webpage with single loaded script', suiteContext, async (context) => {
+        let scriptToMatch = createScriptItemFromInfo('app.js', TEST_SPEC.props.url + 'app.js');
```

Comment:
What do you think of creating an utility to be able to do something like this:
		new LoadedSourcesWizard(context.debugClient).assertSourcesAre(`
			${TEST_SPEC.props.url}/app.js
			${TEST_SPEC.props.url}/otherFile.ts
			`);
(We'll need to figure out how to eventually make the paths work on Mac too)